src/clr-angular/data/datagrid/_datagrid.clarity.scss

```diff
@@ -467,24 +467,25 @@
         }
 
         .datagrid-column-handle {
-          @include clr-no-styles-button();
-          display: block;
           position: absolute;
           width: calc(0.5rem + #{$clr-rem-1px});
-          right: -0.5rem;
+          right: -0.25rem;
           top: -0.25rem;
           cursor: col-resize;
           height: calc(100% + 0.5rem - #{$clr-default-borderwidth});
           z-index: map-get($clr-layers, datagrid-header);
         }
-        .datagrid-column-handle-tracker {
+        .datagrid-column-resize-tracker {
           position: absolute;
-          right: 0;
-          top: $clr-datagrid-column-handle-tracker-top;
+          top: -0.5rem;
```

Comment:
This changed to a hard coded value (which I'm ok if its only ever used for this feature). However, I dont see its removal from the `_variables.datagrid.scss` file.

Reply:
You're right! I will remove it from those variables.